src/components/app-cta/app-cta.scss

```diff
@@ -1,13 +1,17 @@
 .cta {
   height: 344px;
+  background: #008092;
```

Comment:
Is this a new color in the style guide? We should add it globally if we are going to start using it.

Reply:
yes, moved to a variable now in this PR, I guess it should be done in all the others that use this color now too.